app/services/solidus_square/payments/create.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SolidusSquare
+  module Payments
+    class Create < ::SolidusSquare::Base
+      attr_reader :client, :source_id, :amount, :auto_complete
```

Comment:
Can you move the attr_reader under the private section?

Reply:
https://github.com/nebulab/solidus_square/pull/49/commits/6dcd98efc275d3b3cba8faae8338ec2231940fd9